### DISCUSSION TITLE
Topic numbering

### DISCUSSION THREAD

**Discussion by smdita** [dita-ot/dita-ot#3982] [Q&A]
I am working on a customized pdf output (apache fop). I have a bookmap with part>chapter>topics

In the pdf output. The topic titles are coming like part.chapter.topic. I want to get rid of the part number in the topic title.

![image](https://user-images.githubusercontent.com/110420950/182220425-b3727492-2ad0-4352-8c6d-fd300c7057b3.png)

In the above example 1 is part number. 

If I understand correctly, the same numbering is used in toc and bookmarks as well (getNavTitle). If I can fix it in navtitle for the topic then it will also get removed from toc and bookmarks, correct?

 Any suggestions, how I can get rid of the part number?

**Comment by raducoravu** [comment]
@smdita maybe you can post yourself an answer here. From what I remember from our discussion you found the xsl:number in your PDF XSLT customization which produced this counter.

**Comment by smdita** [comment]
<xsl:number format="1.1" count="(//*[ancestor-or-self::*[contains(@class, ' map/topicref bookmap/chapter')]][not(ancestor-or-self::*[contains(@class, ' bookmap/frontmatter ')])])" level="multiple"/>
 

Removes the part number from the counter.

But chapter/section number still getting reset for each part.

**Reply by bing-10** [reply]
> <xsl:number format="1.1" count="(//_[ancestor-or-self::_[contains(@Class, ' map/topicref bookmap/chapter')]][not(ancestor-or-self::*[contains(@Class, ' bookmap/frontmatter ')])])" level="multiple"/>
> 
> Removes the part number from the counter.
> 
> But chapter/section number still getting reset for each part.

@smdita Hello, I am wondering why such counters for chapter and part are not available in map?
